packages/core-data/src/resolvers.js

```diff
@@ -984,3 +984,29 @@ export const getRevision =
 			dispatch.receiveRevisions( kind, name, recordKey, record, query );
 		}
 	};
+
+/**
+ * Requests a specific post type options from the REST API.
+ *
+ * @param {string} postType Post type slug.
+ */
+export const getRegisteredPostMeta =
+	( postType ) =>
+	async ( { dispatch, resolveSelect } ) => {
+		try {
+			const {
+				rest_namespace: restNamespace = 'wp/v2',
+				rest_base: restBase,
+			} = ( await resolveSelect.getPostType( postType ) ) || {};
+			const options = await apiFetch( {
+				path: `${ restNamespace }/${ restBase }/?context=edit`,
+				method: 'OPTIONS',
+			} );
+			dispatch.receiveRegisteredPostMeta(
+				postType,
+				options?.schema?.properties?.meta?.properties
+			);
+		} catch {
+			dispatch.receiveRegisteredPostMeta( postType, false );
```

Comment:
What does `false` mean in this context? What is the purpose of setting the response to a `boolean` value?

Reply:
I have created [this follow-up pull request](https://github.com/WordPress/gutenberg/pull/65450) to handle this. Let's discuss there any potential change.